src/clojure/nrepl/middleware/interruptible_eval.clj

```diff
@@ -39,11 +39,15 @@
         (doto (.setAccessible true))
         (.set reader column))))
 
-(defn- source-logging-pushback-reader
+(defn source-logging-pushback-reader
   [code line column]
   (let [reader (LineNumberingPushbackReader. (StringReader. code))]
-    (when line (set-line! reader (int line)))
-    (when column (set-column! reader (int column)))
+    (try
```

Comment:
This and the other similar instance should have some comments explaining the need for the try/catch, as it's definitely not obvious.